API/Entities/EUser/UserController.cs

```diff
@@ -44,7 +57,16 @@ public async Task<ActionResult<Guid>> Post([FromBody] CreateUserDTO request)
             }
         }
 
+        /// <summary>
+        /// Gets a user by Id
```

Comment:
Domyślnie się bierze po id więc chyba nie musimy tego podkreślać
```suggestion
        /// Gets a User
```